packagedoc/additional_docs/History.tex

```diff
@@ -46,4 +46,10 @@
 \historyversiondate{0.7.5}{03/2024}
 \historychange{- Simplify local configuration loading based on the new version of JsonPreprocessor}
 
+\historyversiondate{0.7.6}{04/2024}
+\historychange{Validates parameter names which are loaded from configuration files:\newline
+- Verify all elements in params['global'] which are set as global variables of Robot run.\newline
+- Parameter names must start with a letter or underscore, followed by letters, digits, or underscores.\newline
+- Ensure that the variable name does not conflict with reserved keywords.}
+
```

Comment:
Hi Son,

this is so much long winded wording and contains too much technical details for an entry in history.

That's really enough:

`Added naming convention check for parameter names.`

To explain the rules behind the naming convention is for the documentation.

Reply:
Hello Holger,

Thank you for your suggestion. I updated